CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# [WIP] openAGI Contributing Guidelines
+
+
+
+openAGI projects accept contributions through GitHub pull requests. This document outlines some of the conventions on development workflow, commit message formatting, contact points, and other resources to make it easier to get your contribution accepted.
+
+
+
+## By contributing to this project you agree to the [MIT License](https://openagi.github.io/tefla/license/). 
+
+## Code of Conduct 
+   Please read the [Code of Conduct](../CODE_OF_CONDUCT.md)
+
+## Support Channels
+
+
+
+The official support channels, for both users and contributors, are:
+
+- GitHub issues: each repository has its own list of issues.
+
+- Slack: join the [openAGI] Slack] (maintainers please add the link to this channel. I am not sure if it already exists) community.
```

Comment:
There's a channel at:  https://openagi.slack.com , interested members of the community can join using this link: [Join OepnAGI Channel](https://join.slack.com/t/openagi/shared_invite/enQtNTEzMDU3NjcwOTQ0LTZlNTg2NTFiNWNiNmJhNzczNjFlZDFiY2I4MjlmYjMzZTVmODk5YWExZWI4OGViZjU0Yjg3ZmU1ZTRkMWRmZTE)